libs/ng-lib/src/lib/scully-content/scully-content.component.ts

```diff
@@ -169,7 +175,10 @@ export class ScullyContentComponent implements OnDestroy, OnInit {
     parent.insertBefore(template.content, this.elm);
     parent.insertBefore(end, this.elm);
     /** upgrade all hrefs to simulated routelinks (in next microtask) */
-    setTimeout(() => this.document.querySelectorAll('[href]').forEach(this.upgradeToRoutelink.bind(this)), 10);
+    setTimeout(
+      () => (this.#localLinksOnly ? parent : this.document).querySelectorAll('[href]').forEach(this.upgradeToRoutelink.bind(this)),
```

Comment:
why use a ternary here? it is not an if/else.
I think wrapping it in a normal if does convey the meaning better?

Reply:
Sure can be changed, usually I go subjectively with such inline code style but agree that for OSS lib to communicate the idea more clearly it can be beneficial to make it more explicit, gonna adjust!